.github/workflows/release-chart.yml

```diff
@@ -2,7 +2,7 @@ name: Release Helm Chart
 on:
   push:
     tags:
-      - "v*"
+      - "*"
```

Comment:
This breaks the new release workflow where a pre-rel-* tag is pushed to create the pre-release workflow as a part of the image publishing step.